example/TestModules/TestModuleAlgo.hs

```diff
@@ -27,8 +31,8 @@ mapP f (x : xs) = f x : mapP f xs
 
 -- non-uniform function with sugared Haskell types
 zip :: [a] -> [b] -> [(a, b)]
-zip []       _        = []
-zip _        []       = []
+zip [] _ = []
+zip _ [] = []
```

Comment:
Since Floskell does not align arguments, the closest we can get is the following.

```haskell
zip :: [a] -> [b] -> [(a, b)]
zip [] _              = []
zip _ []              = []
zip (x : xs) (y : ys) = (x, y) : zip xs ys
```

Currently Floskell does not do that because a padding of 13 Spaces would have to be inserted, but it is configured to insert at most 10 spaces. We could increase that limit. However, Floskell does not distinguish between single-line and multi-line matches like Brittany. Thus, allowing more padding would cause the equal sign to be pushed very far to the right when `do`-blocks are used which could negatively impact readability. For example, I found the following quite difficult to parse.

<details>
<summary>Example</summary>

```haskell
applyGEExp :: Member Fresh r => S.Exp a -> Sem r (S.Exp a)
applyGEExp (S.InfixApp _ e1 qop e2)       = do
  e1' <- applyGEExp e1
  e2' <- applyGEExp e2
  return $ S.InfixApp S.NoSrcSpan e1' qop e2'
applyGEExp (S.NegApp _ expr)              = do
  expr' <- applyGEExp expr
  return $ S.NegApp S.NoSrcSpan expr'
applyGEExp (S.App _ e1 e2)                = do
  e1' <- applyGEExp e1
  e2' <- applyGEExp e2
  return $ S.App S.NoSrcSpan e1' e2'
applyGEExp (S.Lambda _ ps e1)             = do
  e' <- applyGEExp e1
  return $ S.Lambda S.NoSrcSpan ps e'
applyGEExp (S.Let _ bs e1)                = do
  e' <- applyGEExp e1
  return $ S.Let S.NoSrcSpan bs e'
applyGEExp (S.If _ e1 e2 e3)              = do
  e1' <- applyGEExp e1
  e2' <- applyGEExp e2
  e3' <- applyGEExp e3
  return $ S.If S.NoSrcSpan e1' e2' e3'
applyGEExp (S.Case _ e1 alts)             = do
  e' <- applyGEExp e1
  alts' <- applyGEAlts alts
  return $ S.Case S.NoSrcSpan e' alts'
applyGEExp (S.Tuple _ boxed es)           = do
  es' <- mapM applyGEExp es
  return $ S.Tuple S.NoSrcSpan boxed es'
applyGEExp (S.List _ es)                  = do
  es' <- mapM applyGEExp es
  return $ S.List S.NoSrcSpan es'
applyGEExp (S.Paren _ expr)               = do
  expr' <- applyGEExp expr
  return $ S.Paren S.NoSrcSpan expr'
applyGEExp (S.ExpTypeSig _ expr typeExpr) = do
  expr' <- applyGEExp expr
  return $ S.ExpTypeSig S.NoSrcSpan expr' typeExpr
-- Variables, constructors and literals remain unchanged.
applyGEExp e@(S.Var _ _)                  = return e
applyGEExp e@(S.Con _ _)                  = return e
applyGEExp e@(S.Lit _ _)                  = return e
```

</details>

Reply:
I've now increased the limit to 20 spaces (see eeea1e541675a4271d024262853cf3a23769bb39).